tests/integrations/mcs/tso/keyspace_group_manager_test.go

```diff
@@ -248,26 +269,27 @@ func (suite *tsoKeyspaceGroupManagerTestSuite) TestTSOKeyspaceGroupSplit() {
 	re.Equal(uint32(2), kg2.ID)
 	re.Equal([]uint32{222, 333}, kg2.Keyspaces)
 	re.True(kg2.IsSplitTarget())
+	time.Sleep(3 * time.Second)
```

Comment:
Why do we need sleep here, we can use `Eventually` to do the same thing.

Reply:
good catch. It's added for debugging. It should be removed.